### PR TITLE
Get-PnpSiteTemplate - Example for PersistMultiLanguageResources

### PR DESCRIPTION
**Pull request by TashunkoWitko**
There was no example for a usage of PersistMultiLanguageResources in combination with handlers.
It is mandatory to add SupportedUILanguages in handlers as otherwise the multilanguage resources do not get exported properly.

Code reference:
https://github.com/pnp/pnpframework/blob/ac4dc39f564fcaa55e140bd7d778149360a265f7/src/lib/PnP.Framework/Provisioning/ObjectHandlers/Extensions/UserResourceExtensions.cs#L92

### CODE REVIEW

**Comment by yogkumgit** [comment]
@get-itips Please comment.

**Comment by get-itips** [comment]
Hello @TashunkoWitko 
how are you?

Even though the PnP documentation articles are on this repository, I need to kindly redirect you to open PnP cmdlets issues/pull requests [here](https://github.com/PnP/PowerShell/), in that repository, where a member of the PnP community will be able to correct the article/review your PR with your feedback.

The documentation for PnP cmdlets is then synced from that repo into this one.

Thank you!

File: sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPSiteTemplate.md [hunk 1]
```diff
@@ -150,6 +150,15 @@ Get-PnPSiteTemplate -Out template.pnp -ListsToExtract "Title of List One","95c4e
 
 Extracts a provisioning template in Office Open XML from the current web, including only the lists specified by title or ID.
 
+### EXAMPLE 16
+```powershell
+Get-PnPSiteTemplate -Out template.xml -Handlers Fields, ContentTypes, SupportedUILanguages -PersistMultiLanguageResources
+```
+
+Extracts a provisioning template in XML format from the current web including the fields, content types and supported ui languages.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPSiteTemplate.md:158]:
```suggestion
Extracts a provisioning template in XML format from the current web including the fields, content types, and supported UI languages.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please see @get-itips' comment.

**Comment by TashunkoWitko** [comment]
Thanks for the feedback, reopened on https://github.com/pnp/powershell/pull/1372

**Pull request closed without merging**